doc/windows/package-manager/winget/returnCodes.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Exit Codes
+description: WinGet return codes and their meanings
+ms.date: 05/02/2023
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Return Codes
+
+## General Errors
+
+| Hex | Decimal | Description |
```

Comment:
I think it would be good to also add a column for the symbol name. For example
`| 0x8A150101 | -1978334975 | APPINSTALLER_CLI_ERROR_INSTALL_PACKAGE_IN_USE  | Application is currently running. Exit the application then try again.`
The name may help a bit with the meaning, or it could make it easy to look up where it's used in the code

Reply:
Will do